go-controller/pkg/ovn/base_network_controller_pods.go

```diff
@@ -454,6 +454,19 @@ func (bnc *BaseNetworkController) addLogicalPortToNetwork(pod *kapi.Pod, nadName
 		return nil, nil, nil, false, fmt.Errorf("[%s] failed geting expected switch name when adding logical switch port: %v", podDesc, err)
 	}
 
+	if bnc.TopologyType() == ovntypes.LocalnetTopology {
+		netInfo := bnc.GetNetInfo()
+		networkName := netInfo.GetNetworkName()
+		if physicalNetworkName := netInfo.PhysicalNetworkName(); physicalNetworkName != "" {
+			networkName = physicalNetworkName
+		}
```

Comment:
We can call `PyhsicalNetworkName` twice to make it more readable, also maybe this little thing merits a helper.

```golang
if netInfo.PhysicalNetworkName(); != "" {
			networkName = netInfo.PhysicalNetworkName()
		}
```

Reply:
will do the change, but i dont think it merits a helper
we have already helper for `checkBridgeMapping` few lines below

Reply:
Fair enough, just simplify the if statement and we good.

Reply:
did, thx